lang/ruby/avro.gemspec

```diff
@@ -0,0 +1,35 @@
+# -*- encoding: utf-8 -*-
+# stub: avro 1.9.0.pre1 ruby lib
+
+Gem::Specification.new do |s|
```

Comment:
The gemspec is built using Echoe in the [Rakefile](https://github.com/apache/avro/blob/trunk/lang/ruby/Rakefile#L20). Could you merge your changes into there?